lib/libzfs_core/libzfs_core.c

```diff
@@ -1623,6 +1623,24 @@ lzc_pool_checkpoint_discard(const char *pool)
 	return (error);
 }
 
+/*
+ * Load the DDT table for the specified pool.
+ */
+int
+lzc_pool_prefetch_ddt(const char *pool)
+{
+	int error;
+	nvlist_t *result = NULL;
+	nvlist_t *args = fnvlist_alloc();
+
+	error = lzc_ioctl(ZFS_IOC_POOL_PREFETCH_DDT, pool, args, &result);
```

Comment:
All this logic could also apply to BRT.  While it is not a huge deal to add specific IOCTL, my first thought was that zpool_prefetch_type will be passed to kernel as parameter instead, or a mask of needed types, since it may be faster to prefetch them same time.

Reply:
Good point.  Made the ioctl generic `prefetch`